index.js

```diff
@@ -0,0 +1,321 @@
+        if (!lettersCounted[character]) {
+            lettersCounted[character] = 0;
+        }
+        lettersCounted[character]++;
+    }
+    return lettersCounted;
+}
+console.log(countLetters('The quick brown fox jumps over the lazy dog'));
+
+// Write the getAgeDifference function that finds the difference between the youngest and oldest
+// person. Inside of the getAgeDifference function call the getYoungestPerson and
+// getOldestPerson functions that you write yourself.
+
+const peopleArray = [
+    {
+        name: 'Adam',
+        age: 20
+    },
+    {
+        name: 'Amanda',
+        age: 5
+    },
+    {
+        name: 'John',
+        age: 75
+    },
+    {
+        name: 'Dave',
+        age: 15
+    }
+]
+console.log(peopleArray);
+function getYoungestPerson(people) {
+    let theYoungestPerson = people[0].age;
+    for (let i = 0; i < people.length; ++i) {
+        if (people[i].age < theYoungestPerson) {
+            theYoungestPerson = people[i]
+        }
+    }
+    return theYoungestPerson;
+}
+
+function getOldestPerson(people) {
+    let theOldestPerson = people[0].age;
+    for (let i = 0; i < people.length; ++i) {
+        if (people[i].age > theOldestPerson) {
+            theOldestPerson = people[i]
+        }
+    }
+    return theOldestPerson;
+}
+console.log(getOldestPerson(peopleArray))
+function getAgeDifference(people) {
+    return getOldestPerson(people).age - getYoungestPerson(people).age;
+}
+console.log(getAgeDifference(peopleArray)); // 70)
+// Sum of positive
+function positiveSum(arr) {
+    let sum = 0;
+    for (let i = 0; i < arr.length; ++i) {
+        if (arr[i] > 0) {
+            sum = sum + arr[i];
+        }
+    }
+    return sum;
+}
+
+// Square(n) Sum
+
+function squareSum(numbers) {
+    let total = 0;
+    for (let i = 0; i < numbers.length; ++i) {
+        total = total + numbers[i] **2
+    }
+    return total;
+}
+
+console.log(squareSum([1, 2, 2]));
+
+// Abbreviate a Two Word Name
+
+function abbreviationName(name){
+    let nameArray = [name[0]];
+    let abreviation = name.split('');
+    for (let i = 0; i < name.length; ++i) {
+        if(name[i] === " ") {
+            nameArray.push(abreviation[i + 1])
+        }
+    }
+    return nameArray.join('.').toUpperCase();
+}
+
+console.log(abbreviationName('Sam Harris'));
+console.log(abbreviationName('patrick feeney'));
+
+// Counting sheep...
+function countSheep(arrayOfSheep) {
+    let present = true;
+    let sheep = 0;
+    for (let i = 0; i < arrayOfSheep.length; ++i) {
+        if (arrayOfSheep[i] === present) {
+           sheep++;
+        }
+    }
+    return sheep;
+}
+
+console.log(countSheep([true,  true,  true,  false,
+    true,  true,  true,  true ,
+    true,  false, true,  false,
+    true,  false, false, true ,
+    true,  true,  true,  true ,
+    false, false, true,  true]));
+
+// Convert number to reversed array of digits
+
+function digitize(number) {
+    return Array.from(String(number), Number).reverse()
```

Comment:
Please explain to me in detail what happens in `Array.from(String(number), Number)`. Did you figure out this solution yourself?